tests/tf-manifests/rhcs/clusters/rosa-hcp/variables.tf

```diff
@@ -187,4 +187,9 @@ variable "disable_waiting_in_destroy" {
 variable "full_resources" {
   type    = bool
   default = false
+}
+
+variable "worker_disk_size" {
```

Comment:
Please also update the golang struct for this new variable

Reply:
It does already exist as it was used with Classic